### ISSUE TITLE
Feature: Dynamic token lists

### ISSUE THREAD

**Issue by lllwvlvwlll** [CityOfZion/neon-wallet#1358]
Currently, neon wallet relies on a manual list of nep5 script hashes to maintain tokens, which requires users to download and reinstall a new wallet version every time there is a new token sale etc...  This proposal is to remove the static list and replace it with a periodic request to this repo where a file is maintained containing the list of script hashes.

https://github.com/CityOfZion/neo-tokens/blob/master/tokenList.json

**Comment by tester2011** [comment]
Surely the used can add them manually using the "Manage Neon Settins->Manage Token" setting? It's a simple task to add the script hash. Waiting for a new version of the wallet seems a bit extreme. Am I missing spmething?

On the question of which -= is there a list somewhere that shows the current NEP-5 tokens that have been added? I installed v 0.2.6 and them went about adding the ONT hash, only to find it was already there. On the other hand, I had to add the NK hash, as it wasn't there.

Cheers,

Mike.
